samples/client/petstore/typescript-angular2/npm/api/PetApi.ts

```diff
@@ -258,7 +258,7 @@ export class PetApi {
             throw new Error('Required parameter petId was null or undefined when calling deletePet.');
         }
 
-        headers.set('api_key', String(apiKey));
+            headers.set('api_key', String(apiKey));
```

Comment:
Several additional spaces added to this line. Not sure if it's related to this PR.